WordPress/gutenberg
# Format Library: Add alt edit field to inline image

Fixes #63323
Related to #28813

## What?

This PR makes it possible to change the alt text in the inline image popover.

![image](https://github.com/user-attachments/assets/579be804-4e48-4b32-85d3-8359a6a532d9)

## Why?

If the media already has alt text set, the alt attribute will be applied to the inserted inline image. However, if you want to change the alt attribute, you have no choice but to change the alt of the original media and then reinsert it, or change the value directly via the code editor.

## How?

Add a text area to the popover to change the alt text.

## Testing Instructions

- Add an inline image.
- Select the image.
- Change alt text and click the Apply button.
- If you check the HTML, you'll see that the alt text has changed.

### Testing Instructions for Keyboard

You should be able to change the alt text using just the keyboard.

## Screenshots or screencast 

https://github.com/user-attachments/assets/73a495ad-91a4-4bae-a49e-a1efb3c2bc97
